trace/theme_manager.py

```diff
@@ -0,0 +1,391 @@
+            IconColors.WARNING: "#ff8c00",  # Orange for warnings
+            IconColors.ERROR: "#d13438",  # Red for errors
+            IconColors.DISABLED: "#999999",  # Light gray for disabled
+        }
+
+        self.dark_icon_colors: IconColorDict = {
+            IconColors.PRIMARY: "#ffffff",  # White for primary icons
+            IconColors.SECONDARY: "#cccccc",  # Light gray for secondary icons
+            IconColors.ACCENT: "#0078d4",  # Blue for accent colors
+            IconColors.SUCCESS: "#107c10",  # Green for success
+            IconColors.WARNING: "#ff8c00",  # Orange for warnings
+            IconColors.ERROR: "#d13438",  # Red for errors
+            IconColors.DISABLED: "#666666",  # Dark gray for disabled
+        }
+
+    def _load_stylesheet(self, stylesheet_path: str | None) -> str:
+        """
+        Load a QSS stylesheet from file.
+
+        Parameters
+        ----------
+        stylesheet_path : str | None
+            Path to the QSS stylesheet file, or None to return empty string.
+
+        Returns
+        -------
+        str
+            The stylesheet content, or empty string if file cannot be loaded.
+
+        Examples
+        --------
+        >>> stylesheet = theme_manager._load_stylesheet("styles/dark.qss")
+        """
+        if not stylesheet_path:
+            return ""
+
+        try:
+            with open(stylesheet_path, "r", encoding="utf-8") as file:
+                return file.read()
+        except FileNotFoundError:
+            print(f"Warning: Stylesheet file not found: {stylesheet_path}")
+            return ""
+        except UnicodeDecodeError:
+            print(f"Warning: Could not decode stylesheet file: {stylesheet_path}")
+            try:
+                # Try with different encoding
+                with open(stylesheet_path, "r", encoding="latin-1") as file:
+                    return file.read()
+            except Exception as e:
+                print(f"Warning: Failed to load stylesheet {stylesheet_path}: {e}")
+                return ""
+        except Exception as e:
+            print(f"Warning: Failed to load stylesheet {stylesheet_path}: {e}")
+            return ""
+
+    def set_theme(self, theme: Theme) -> None:
+        """
+        Set the application theme.
+
+        Parameters
+        ----------
+        theme : Theme
+            The theme to apply (Theme.LIGHT or Theme.DARK).
+
+        Examples
+        --------
+        >>> theme_manager.set_theme(Theme.DARK)
+        >>> theme_manager.set_theme(Theme.LIGHT)
+        """
+        self.current_theme = theme
+
+        if theme == Theme.DARK:
+            self.app.setPalette(self.dark_palette)
+            stylesheet = self._load_stylesheet(self.dark_stylesheet_path)
+        else:
+            self.app.setPalette(self.light_palette)
+            stylesheet = self._load_stylesheet(self.light_stylesheet_path)
+
+        self.app.setStyleSheet(stylesheet)
```

Comment:
```suggestion
        self.app.main_window.setStyleSheet(stylesheet)
```

`QApplication.setStyleSheet()` wasn't working for me on RHEL9 servers. Instead, using `QMainWindow.setStyleSheet()` was working for me. This fix let me change stylesheets.